src/coreComponents/physicsSolvers/multiphysics/poromechanicsKernels/SinglePhasePoromechanicsEFEM.hpp

```diff
@@ -354,23 +362,21 @@ struct StateUpdateKernel
       real64 dHydraulicAperture_dNormalJump = 0.0;
       real64 dHydraulicAperture_dNormalTraction = 0.0;
       hydraulicAperture[k] = contactWrapper.computeHydraulicAperture( aperture[k],
-                                                                      fractureTraction[k][0],
+                                                                      fractureContactTraction[k][0],
                                                                       dHydraulicAperture_dNormalJump,
                                                                       dHydraulicAperture_dNormalTraction );
 
       deltaVolume[k] = hydraulicAperture[k] * area[k] - volume[k];
 
-      // traction on the fracture to include the pressure contribution
-      fractureTraction[k][0] -= pressure[k];
-      dFractureTraction_dPressure[k] = -1.0;
-
       real64 const jump[3] = LVARRAY_TENSOROPS_INIT_LOCAL_3 ( dispJump[k] );
-      real64 const traction[3] = LVARRAY_TENSOROPS_INIT_LOCAL_3 ( fractureTraction[k] );
+      real64 const contactTraction[3] = LVARRAY_TENSOROPS_INIT_LOCAL_3 ( fractureContactTraction[k] );
 
+      // all perm update models below should need contact traction instead of total traction 
+      // (total traction is combined forces of fluid pressure and contact traction)
       porousMaterialWrapper.updateStateFromPressureApertureJumpAndTraction( k, 0, pressure[k],
                                                                             oldHydraulicAperture[k], hydraulicAperture[k],
                                                                             dHydraulicAperture_dNormalJump,
-                                                                            jump, traction );
```

Comment:
in the previous version, the total traction is passed to the porousMaterialWrapper to update perm. I think only contact traction is needed here. I corrected it here but it may change the results from Integrated Tests. @CusiniM @paveltomin

Reply:
Now the traction in ContactFields.hpp only represents the contact traction which will be rendered in paraview. Like total stress and effective stress in the rock matrix, user has to do their own calculation to get total force on the fracture.  I think this will make the naming more consistent with the data stored here. @CusiniM @paveltomin

Reply:
I would call it `effectiveTraction` but yes, this is how it should be IMO. It's also how it is for the other contact solver so it's nice to be consistent.